Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.11
+FROM node:16.13
```

Comment:
did you build the docker image to verify?

Reply:
we should only specify the major version of nodejs here and let docker pull any latest tags